Makefile

```diff
@@ -8,16 +8,10 @@ FS_MODULES=/usr/lib/freeswitch/mod
 SHELL := /bin/bash
 PROC?=$(shell uname -m)
 
-CC=gcc 
 CFLAGS=-fPIC -O3 -fomit-frame-pointer -fno-exceptions -Wall -std=c99 -pedantic
-ifeq (${PROC},x86_64)
```

Comment:
Sounds about right, as far as i know you cant really crosscompile on arm anyway.
x86_64 and x86 are indeed the most prominent example of natively working crosscompilation.